torch/nn/modules/transformer.py

```diff
@@ -375,6 +375,100 @@ def forward(self, tgt, memory, tgt_mask=None, memory_mask=None,
         return tgt
 
 
+class MultiheadAttentionInProjection(Module):
+    r"""
+    Args:
+        in_embed_dim (int): Input embedding dimension
+        num_heads (int): Number of parallel attention heads.
+        out_embed_dim (int, optional): Output embedding dimension. If not
+            provided, then set to `in_embed_dim`. Must be divisible by
+            `num_heads`.
+    """
+    def __init__(self, in_embed_dim, num_heads, out_embed_dim=None):
+        # TODO: Bias??
```

Comment:
Should we include the ability to enable and disable a bias parameter? Would be simple enough to add an optional `bias` parameter.

Reply:
Not for now. It's always easier to add codes than removing.

Reply:
Sounds good.